### ISSUE TITLE
HomeKit scene sending multiple requests doesn't work properly

### ISSUE THREAD

**Issue by marcotti71** [kiwi-cam/homebridge-broadlink-rm#440]
Hi,
I created two scenes in HomeKit that requests one to open, the other to close, 4 devices through my broadlink RM4 Pro S.
Each time I call one of the two scenes it "forgets" to turn on (or off) one or two devices, not necessarily the same ones.
Very rarely it opens (or closes) all of them.
If I turn them on or off manually one by one, it works like a charm.
The devices are:
- An Onkyo receiver set as "TV" sub-type "Receiver" (same IR hex code to turn it on or off)
- An Epson projector set as "TV" (two different IR hex codes to turn it on or off, to turn it off need to send the hex code twice, with one second delay)
- A rollable screen set as "window-covering" (two different RF hex codes to open or close it)
- An "HDMI Matrix" (to split the audio from the video signal coming from an HDMI source) set as "switch" (same IR hex code to turn it on or off)

I suspect that the RM4 cannot handle too many inputs at the same time.
If this is the case, is there a way to "delay" somehow the trigger of each command adding a "pause" ahead of the command instruction?

Did any of you encountered the same issue?

HomeKit hub: Apple TV 4K - tvOS 15.2

**Comment by marcotti71** [comment]
Forgot to add: in the HomeBridge log (as in the HomeKit UI) it looks like everything is working fine, but the devices actually do not switch

**Comment by marcotti71** [comment]
Update: I found a workaround to force it working.
Since the "pause" instruction is discarded if added as first instruction of the command (don't know if by HomeKit, by HomeBridge or by the plugin itself), I started my command with a **fake hex code ("1234ab")**, then I added the "pause" and finally the real hex code. E.g.:
`{
	"name": "My Device",
	"type": "switch",
	"data": {
		"on": [
			    {
				"data": "1234ab",
				"pause": 1
			    },
			    {
				"data": "ON_HEX_CODE"
			    }
		],
		"off": [
			    {
				"data": "1234ab",
				"pause": 1
			    },
			    {
				"data": "OFF_HEX_CODE"
			    }
		]
	}
}`

I chose the sequence I wanted to have and (leaving the first device of my sequence unchanged) I applied the configuration above with a increasing pause for any other device (1 sec for the second device, 2 secs for the third and 3 secs for the fourth... I tried to go with 0.5 secs lag steps, but it was still too quick).

Clearly it is a workaround, but I hope it will help whoever will have the same issue

Cheers
m
